Expensify/App
# add home page structure to Help Site

cc @michelle-thompson

### Details

This PR only includes the structure for the homepage and fixes the styles for the homepage and hub pages on large screens. The site's content is still [WIP](https://expensify.slack.com/archives/C02QSAC6BJ8/p1658787454404069), but once the content is done, we're just going to replace the text in the respective HTML or Markdown files and can be done by someone of the resource management team.

### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/216731

### Tests


0. (you need ruby and bundler installed to do this, more info [here](https://github.com/Expensify/App/tree/main/docs#pre-requisites))
1. Run:

    ```bash
    cd docs
    bundle install
    bundle exec jekyll serve
    ```
2. Navigate to http://localhost:4000/main and verify the structure of the homepage is displayed as in the [design document mockups](https://docs.google.com/document/d/1dyJcO08hFD7XVhX92cMhay1qGp3DOK6z3RlelLA9b_4/edit?pli=1#bookmark=id.hiqxqfv66xk2) or the screenshots attached at the end.

### QA Steps


Go to https://help.expensify.com/main and verify the structure of the homepage is displayed (as in the screenshots)

### Screenshots


#### Web


![localhost_4000_main](https://user-images.githubusercontent.com/6829422/181097417-3236e08d-e48b-4b05-bc57-2edc731fc239.png)

Big screen:
<img width="1792" alt="Screen Shot 2022-07-26 at 14 38 20" src="https://user-images.githubusercontent.com/6829422/181097505-75096fbc-4e0b-49a3-81f9-61daee30ccf6.png">

#### Mobile Web

<img width="480" alt="Screen Shot 2022-07-26 at 14 38 20" src="https://user-images.githubusercontent.com/6829422/181094331-b407b456-2e08-451e-b387-10bc8770bc76.png">
